core/domain/user_services.py

```diff
@@ -83,6 +84,9 @@ class UserSettings(python_utils.OBJECT):
         preferred_site_language_code: str or None. System language preference.
         preferred_audio_language_code: str or None. Audio language preference.
         pin: str or None. The PIN of the user's profile for android.
+        display_alias: str or None. Display name of a user who is logged
+            into the Android app. None when the request in coming from web
```

Comment:
in --> is

or 

when the request is coming in from the Web

Reply:
Done.